auth/service.go

```diff
@@ -145,6 +167,21 @@ func (svc service) RetrieveKey(ctx context.Context, token, id string) (Key, erro
 }
 
 func (svc service) Identify(ctx context.Context, token string) (Key, error) {
+	if strings.HasPrefix(token, "pat"+"_") {
```

Comment:
`Identify` is contains `IdentifyPAT`, so I think we can have only `Authnetication` gRPC enough. 
Please check and give your comments